### PR TITLE
GEOMESA-1132 Exporting avro files through tools

### PR DESCRIPTION
**Pull request by elahrvivaz**
Signed-off-by: Emilio Lahr-Vivaz <elahrvivaz@ccri.com>

### CODE REVIEW

File: geomesa-tools/src/main/scala/org/locationtech/geomesa/tools/commands/ExportCommand.scala [hunk 1]
```diff
@@ -120,11 +120,11 @@ class ExportCommand(parent: JCommander) extends CommandWithCatalog(parent) with
 object ExportCommand {
   @Parameters(commandDescription = "Export a GeoMesa feature")
   class ExportParameters extends OptionalCqlFilterParameters {
-    @Parameter(names = Array("-F", "--format"), description = "Format to export (csv|tsv|gml|json|shp|bin)")
+    @Parameter(names = Array("-F", "--format"), description = "Format to export (csv|tsv|gml|json|shp|bin|avro)")
     var format: String = "csv"
 
-    @Parameter(names = Array("-m", "--max-features"), description = "Maximum number of features to return. default: Long.MaxValue")
-    var maxFeatures: Integer = Int.MaxValue
+    @Parameter(names = Array("-m", "--max-features"), description = "Maximum number of features to return. default: Unlimited")
```
Comment (jahhulbert-ccri) [inline geomesa-tools/src/main/scala/org/locationtech/geomesa/tools/commands/ExportCommand.scala]: Is it actually Int.MaxValue because of geotools?

**Comment by jahhulbert-ccri** [comment]
looks good. i'll verify one on the command line.

File: geomesa-tools/src/main/scala/org/locationtech/geomesa/tools/commands/ExportCommand.scala (hunk 1 above)
Comment (elahrvivaz) [inline geomesa-tools/src/main/scala/org/locationtech/geomesa/tools/commands/ExportCommand.scala]: I think that if max features isn't set, we'll just keep returning values forever... if it is set, then it has to fit inside an Int. I could be wrong about that...

Comment (jahhulbert-ccri) [inline geomesa-tools/src/main/scala/org/locationtech/geomesa/tools/commands/ExportCommand.scala]: I really hope its unlimited...we should PR geotools for using a long #bigdata

**Comment by jahhulbert-ccri** [comment]
7c295d68bad92291e4260273134219dd0f938faf

**Pull request closed without merging**